docs/DeveloperGuide.md

```diff
@@ -165,6 +165,9 @@ An `Endpoint` contains the following attributes:
 * Before an API call is made, the `Response` object will be empty
 * Only when a Request#executeTimed(request) is called will a `Response` contain information about the API call response
 
+The following activity diagram summarizes what happens when a user executes a add command:
```

Comment:
Using `an add` might sound more correct here.

Reply:
The grind for comments is real

Reply:
O man